Gemfile.lock

```diff
@@ -137,6 +139,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-19
+  x86_64-linux
```

Comment:
Haven't seen this before... both are needed?

Reply:
I guess this is new for Nokogiri as of 1.11.0
> As of v1.11.0, Nokogiri ships pre-compiled, "native" gems for the following platforms:
...
If you're on a supported platform, either gem install or bundle install should install a native gem without any additional action on your part. This installation should only take a few seconds, and your output should look something like:

```
$ gem install nokogiri
Fetching nokogiri-1.11.0-x86_64-linux.gem
Successfully installed nokogiri-1.11.0-x86_64-linux
1 gem installed
```

source: https://nokogiri.org/tutorials/installing_nokogiri.html#supported-platforms

Reply:
I'm also guessing Mac for local development and linux for CI/CD

Reply:
Right... what I was wondering is if we should then break it into having the Linux version in the Production/Test group, and the Darwin in the Development group.
It might just complicate things, so I'm going to mark resolved.